### PR TITLE
show list items and add local storage

### PR DESCRIPTION
**Pull request by TuncAlt**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,16 +1,21 @@
-import { useState } from "react";
+// import { useState } from "react";
```
Comment (marcbendixen) [inline src/App.js:1]: If not in use, you could remove this line.

File: src/App.js [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
+
 
 function App() {
-  const [activities, setActivities] = useState([]);
+  const [activities, setActivities] = useLocalStorageState("activities", { defaultValue: [] });
 
   function handleAddActivity(newActivity) {
-    setActivities((activities) => [...activities, newActivity]);
+    setActivities([...activities, newActivity]);
     console.log(activities);
```
Comment (marcbendixen) [inline src/App.js:12]: Normally `console.log()` shouldn't be part of a pull request.

File: src/components/List.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+        return  <li key={activity.id} 
+        goodweather={activity.goodweather}
+        >{activity.name} - is good for outdoor: {activity.goodweather ? "Yes" : "No"}</li>
+    })
+  
+return (
+        <div>
+            <ul>{listItems}</ul>
+        </div>
+)}
```
Comment (marcbendixen) [inline src/components/List.js:14]: Another way could be to map directly inside the return:

```suggestion
export function List({ activities }) {
  return (
    <div>
      <ul>
        {activities.map((activity) => {
          return (
            <li key={activity.id} goodweather={activity.goodweather}>
              {activity.name} - is good for outdoor:{" "}
              {activity.goodweather ? "Yes" : "No"}
            </li>
          );
        })}
      </ul>
    </div>
  );
}
```

**Review by marcbendixen — COMMENTED** [review]
Well done 🚀 
I made a few comments

File: src/components/List.js (hunk 3 above)
Comment (TuncAlt) [reply src/components/List.js:14]: thx

**Pull request merged**